cub/cub/device/dispatch/dispatch_reduce.cuh

```diff
@@ -148,7 +148,7 @@ template <typename InputIteratorT,
             InitT,
             AccumT,
             TransformOpT>,
-          typename KernelLauncherFactory = detail::TripleChevronFactory>
+          typename KernelLauncherFactory = CUB_DETAIL_DEFAULT_KERNEL_LAUNCHER>
```

Comment:
This is a good idea! Could you please apply the change consistently to all kernel launcher default arguments. There should be about 12. Maybe in a separate PR and then rebase? Thx!

Reply:
I disagree. This change will increase inconsistency in CUB and we already have too much of that. It's better to make a consistent change atomically. Also, a separate PR will take 10min.